Modules/Managers/PathManager.m

```diff
@@ -264,139 +269,37 @@ function new_alias_GUI(obj,varargin)
             if ~isempty(varargin)&&ischar(varargin{1})
                 default_name = varargin{1};
             end
-            name = inputdlg('Name:','New Path: Name',[1 75],{default_name});
-            if isempty(name) % User aborted, no worries
-                return
-            end
-            name = name{1};
-            obj.validate_name(name);
-            f = figure('name',sprintf('Instructions for "%s": Select instruction type',name),'IntegerHandle','off','menu','none',...
-                'toolbar','none','visible','off','units','characters');
-            f.UserData.uiwait = false;
-            [lbox,ok] = listbox(f,'OK','string',{},'min',0,'max',0);
-            delete(ok)
-            % Get module (optional)
-            for i = 1:length(obj.module_types)
-                package = ['+' Modules.(obj.module_types{i}).modules_package];
-                mod_name = strsplit(obj.module_types{i},'.'); mod_name = mod_name{end};
-                parent_menu = uimenu(f,'Text',mod_name);
-                Base.Manager.getAvailModules(package,parent_menu,@obj.selected,@(~)false);
-            end
-            uimenu(f,'Text','Function (no module)','callback',@obj.selected,'tag','','UserData','');
-            f.UserData.module = '';
-            
-            instructions = struct('module_name',{},'module_inputs',{},'method_name',{},'method_inputs',{});
+            default_code = '';
+            lock_name = false;
             while true
-                temp = obj.UIgetInstruction(f);
-                if ~isempty(temp)
-                    instructions(end+1) = temp;
-                end
-                if ~isvalid(f) % User aborted
-                    return
-                end
-                % Update listbox strings
-                lbox.String = obj.get_string_instructions(instructions);
-                answer = questdlg('Add another instruction?',mfilename,'Yes','No','Yes');
-                if ~strcmp('Yes',answer)
+                % Prepare code for path
+                pre = ['% Write the relevant code to transition to relevant path.' newline,...
+                       '% The function name will become the path name.', newline,...
+                       '%     You can use CommandCenter''s "**" menu to help with modules' newline];
+                [code,name] = uigetcode(obj.temp_file,default_name,pre,'',default_code,~lock_name);
+                if strcmp(name,'PATH_NAME')
+                    resp = questdlg('Did you mean to keep the "PATH_NAME" default name?','Yes','No','No');
+                    if strcmp(resp,'Yes')
+                        break
+                    end
+                else
                     break
                 end
             end
-            delete(f);
-            if ~isempty(instructions)
-                obj.new_path(name,instructions)
+            % Save
+            if ~isempty(code)
+                obj.new_path(name,code,false)
             end
```

Comment:
keeps logic same as before; if empty, silently fail. Up for suggestions here though.

Reply:
do you think it's worth a warning that it's empty?